File: app/src/main/kotlin/br/com/sailboat/elseapp/view/ui/view_model/DrugListViewModel.java
package br.com.sailboat.elseapp.view.ui.view_model;

import java.util.ArrayList;
import java.util.List;

import br.com.sailboat.elseapp.base.BaseViewModel;
import br.com.sailboat.elseapp.model.Drug;

public class DrugListViewModel extends BaseViewModel {

    private transient final List<Drug> drugList;

    public DrugListViewModel() {
        this.drugList = new ArrayList<>();
    }

    public List<Drug> getDrugList() {
        return drugList;
    }

}

File: app/src/main/kotlin/br/com/sailboat/elseapp/view/drug/list/view_model/DrugListViewModel.kt
package br.com.sailboat.elseapp.view.drug.list.view_model

import br.com.sailboat.elseapp.base.BaseViewModel
import br.com.sailboat.elseapp.model.Drug
import java.util.*

class DrugListViewModel : BaseViewModel() {

    @Transient val drugList: MutableList<Drug>

    init {
        this.drugList = ArrayList<Drug>()
    }

}
